catalog/core/catalog-core-api-impl/src/main/java/org/codice/alliance/catalog/core/api/impl/types/IsrAttributes.java

```diff
@@ -0,0 +1,363 @@
+                true /* indexed */,
+                true /* stored */,
+                false /* tokenized */,
+                false /* multivalued */,
+                BasicTypes.LONG_TYPE));
+        DESCRIPTORS.add(new AttributeDescriptorImpl(FRQUENCY_HERTZ,
+                true /* indexed */,
+                true /* stored */,
+                false /* tokenized */,
+                false /* multivalued */,
+                BasicTypes.FLOAT_TYPE));
+        DESCRIPTORS.add(new AttributeDescriptorImpl(IMAGE_ID,
+                true /* indexed */,
+                true /* stored */,
+                true /* tokenized */,
+                false /* multivalued */,
+                BasicTypes.STRING_TYPE));
+        DESCRIPTORS.add(new AttributeDescriptorImpl(JC3IEDM_ID,
+                true /* indexed */,
+                true /* stored */,
+                true /* tokenized */,
+                false /* multivalued */,
+                BasicTypes.STRING_TYPE));
+        DESCRIPTORS.add(new AttributeDescriptorImpl(MISSION_ID,
+                true /* indexed */,
+                true /* stored */,
+                true /* tokenized */,
+                false /* multivalued */,
+                BasicTypes.STRING_TYPE));
+        DESCRIPTORS.add(new AttributeDescriptorImpl(NATO_REPORTING_CODE,
+                true /* indexed */,
+                true /* stored */,
+                true /* tokenized */,
+                false /* multivalued */,
+                BasicTypes.STRING_TYPE));
+        DESCRIPTORS.add(new AttributeDescriptorImpl(NATIONAL_IMAGERY_INTERPRETABILITY_RATING_SCALE,
+                true /* indexed */,
+                true /* stored */,
+                false /* tokenized */,
+                false /* multivalued */,
+                BasicTypes.INTEGER_TYPE));
+        DESCRIPTORS.add(new AttributeDescriptorImpl(ORGANIZATIONAL_UNIT,
+                true /* indexed */,
+                true /* stored */,
+                true /* tokenized */,
+                true /* multivalued */,
+                BasicTypes.STRING_TYPE));
+        DESCRIPTORS.add(new AttributeDescriptorImpl(ORIGINAL_SOURCE,
+                true /* indexed */,
+                true /* stored */,
+                true /* tokenized */,
+                false /* multivalued */,
+                BasicTypes.STRING_TYPE));
+        DESCRIPTORS.add(new AttributeDescriptorImpl(PLATFORM_ID,
+                true /* indexed */,
+                true /* stored */,
+                true /* tokenized */,
+                false /* multivalued */,
+                BasicTypes.STRING_TYPE));
+        DESCRIPTORS.add(new AttributeDescriptorImpl(PLATFORM_NAME,
+                true /* indexed */,
+                true /* stored */,
+                true /* tokenized */,
+                false /* multivalued */,
+                BasicTypes.STRING_TYPE));
+        DESCRIPTORS.add(new AttributeDescriptorImpl(REPORT_ENTITY_ALIAS,
```

Comment:
Multiple reports can be included, or can they just have multiple names? The alias and name attributes are both multivalued whereas the other report-associated attributes are not.